### ISSUE TITLE
Windows compatibility

### ISSUE THREAD

**Issue by amitmerchant1990** [paulirish/git-recent#7]
Hi @paulirish 

I have installed the module on my machine running Windows 10 and got the following:
![git-recent](https://cloud.githubusercontent.com/assets/3647841/17471982/b87d769e-5d65-11e6-887c-9cafcfd261f8.JPG)
I think something is missing while running it on the Windows.

**Comment by scottoffen** [comment]
I'm seeing the same issue on Windows 7. I just installed version 1.0.1.

`column` is a bash command, not available on Windows.

**Comment by paulirish** [comment]
`less` exists on windows, right?

open to suggestions to replace the column call. I don't know much about bash scripting against windows. :/

**Comment by amitmerchant1990** [comment]
@paulirish Can we use space character ' ' instead of `column` command? as it seems there is no equivalent command similar to `column` on Windows.

**Comment by daniel-liuzzi** [comment]
You need [Git BASH](https://git-for-windows.github.io/#bash) to be able to run BASH commands on Windows. For example, [GitHub Desktop](https://desktop.github.com/) sets this up for you –among other things– and should fix the issue you're facing. I have it installed and `column`, `less`, `git recent`, etc. all run just fine:

```
> column --help

Usage:
column [options] [<file>...]

Columnate lists.

Options:
-c, --columns <width>    width of output in number of characters
-t, --table              create a table
-s, --separator <string> possible table delimiters
-o, --output-separator <string>
                         columns separator for table output; default is two spaces
-x, --fillrows           fill rows before columns

-h, --help     display this help and exit
-V, --version  output version information and exit

For more details see column(1).
>
```

This is on Windows 10.

**Comment by amitmerchant1990** [comment]
Thanks @dliuzzi. This worked for me as well.
![git_recent_update](https://cloud.githubusercontent.com/assets/3647841/17586756/3702e5ee-5fe1-11e6-98f3-cf1277615047.JPG)

**Comment by daniel-liuzzi** [comment]
Glad to help! By the way I notice you're seeing the same funky format I see. See #8.

**Comment by paulirish** [comment]
I don't think we'll be able to get decent rendering without `column` so i'm open to having `git bash` or w/e required for windows users...  @amitmerchant1990 if you got time, can you do a PR to add that to the readme?

**Comment by amitmerchant1990** [comment]
Sure @paulirish

**Comment by paulirish** [comment]
fixed by d1ed7ac35eef4658d1c4c963adb72eff8bc21013
